.travis.yml

```diff
@@ -1,7 +1,4 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
-  - "4"
```

Comment:
Whoa, dropping support for 4.x, which is still LTS, and requiring only 6.x, which is not LTS yet. I don't think users will like this.

It's not necessarily a blocker, but we must come up with a reasonable way how to support users running on pre-v6.x versions. Perhaps publish the new version under the `next` tag, so that `npm install loopback-context` keep installing the current `stable` version line `1.x`?